src/utils/get-element.js

```diff
@@ -0,0 +1,3 @@
+const getElement = () => (typeof Element === 'undefined' ? () => {} : Element);
+
+export default getElement;
```

Comment:
react-modal uses this: https://github.com/reactjs/react-modal/blob/master/src/helpers/safeHTMLElement.js

We can do the same maybe

Reply:
thanks, [4e261485b9ffe6ffcf75e49f60441c2a86363816](https://github.com/commercetools/ui-kit/pull/692/commits/4e261485b9ffe6ffcf75e49f60441c2a86363816)